.github/workflows/docker-image.yml

```
@@ -4,296 +4,283 @@ on:
+    needs: build-and-push
     runs-on: ubuntu-latest
+    if: github.event_name != 'pull_request'
     permissions:
       contents: read
-      packages: write
+      security-events: write
+    strategy:
+      fail-fast: false
+      matrix:
+        platform: [linux/amd64, linux/arm64]
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)